src/incentive-details.tsx

```diff
@@ -80,7 +80,7 @@ function formatAmount(amount: number, amount_type: AmountType) {
       return `$${amount.toLocaleString()}`;
     }
   } else {
-    return amount.toString();
+    return `$${amount.toLocaleString()}`;
```

Comment:
I think for solar this is supposed to be a "30%" number, at least if we have parity with the www version? I forget what our precedence logic is for percent vs representative amount but I think the representative amounts were only intended for totals?

Reply:
v0 returns the solar tax credit like this:
```
 "amount": 5130,
 "amount_type": "solar",
 "item_type": "solar_tax_credit",
 "representative_amount": 0,
```

The percentage isn't in there anywhere.

www actually hardcodes "30%": https://github.com/rewiringamerica/rewiringamerica.org/blob/61c2fd7eae155a7ce09506bb27a1be65079b5e41/www-rewiringamerica-org/src/components/IraCalculator/Tables/index.tsx#L90

v1 (and thus the new embed) returns it as a `0.3 / "percent"` amount with the approximation in the representative amount, and displays it as "30%". I think that's the right end state.